Makefile

```diff
@@ -58,8 +58,15 @@ checksum:
 clean:
 	@rm -rf $(BINDIR) ./_dist
 
+GIT_BASEDIR    = $(shell git rev-parse --show-toplevel 2>/dev/null)
+ifneq ($(GIT_BASEDIR),)
+	LDFLAGS += -X github.com/Azure/acs-engine/pkg/test.JUnitOutDir=${GIT_BASEDIR}/test/junit
+endif
+
 test: test-style
-	go test -v $(GOFILES)
+	go get github.com/onsi/ginkgo/ginkgo
+	go get github.com/onsi/gomega
```

Comment:
right what im getting at is that msft/acs-engine is a distribution image not a developmental image. its a separation of concerns that cleans up how we manage ci and the overall development environment as a whole. the development image in question is definitely public and can be used by anyone to build and generate everythign they need within the project regardless of their environment.